index.html

```diff
@@ -38,6 +38,17 @@ <h2>Comparing Triangle-Collapse and Edge-Collapse</h2>
       });
     </script>
 
+    <h3 class="left">Triangle-Collapse</h3>
+    <h3 class="right">Edge-Collapse</h3>
+    <div class="img-cmp">
+      <img src="assets/img/decim_edge_poly.jpg" draggable="false">
+      <div class="cmp">
+        <img src="assets/img/decim_tri_poly.jpg" draggable="false">
+      </div>
+      <div class="after-cmp"></div>
+    </div>
+    <p>With the bottom plate we can clearly see how Triangle-Collapse quickly starts eating away even at large overall shapes. In order to get this under control, more &quot;globalist&quot; error metrics would need to be employed - but Edge-Collapse would most certainly similarly benefit from this. On the other hand, this is actually the only case where Edge-Collapse also ends up damaging normals (by flipping the triangle vertex-order). Still, it's far less pronounced as Triangle-Collapse. Ignoring all the issues, I think I prefer Edge-Collapse here over Triangle-Collapse, as it resulted in much smoother looking decimation. I expected the thin paper geometry to cause issues, but this Fortunately didn't occur.</p>
```

Comment:
`but this Fortunately didn't occur.` -> `but this fortunately didn't occur.` fortunately klein